src/tests/unit_tests/agents/test_critic_agent.py

```diff
@@ -19,8 +21,8 @@
 """  # noqa: E501
 
 
-def test_evaluation_matrices():
-    llm = OpenAI(openai_api_key=cfg.OPENAI_API_KEY, temperature=0)
+def test_evaluation_matrices_successful(get_llm):
```

Comment:
test_evaluation_matrices_successful -> test_evaluation_matrices_succeeds

(global search/replace function names like this)